upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,17 @@
 == Migration Changes
 
+=== Migrating to 11.0.0
+
+==== LDAP no-import bugfix
+
+In the previous {project_name} version, when LDAP provider was configured with `Import Users` OFF, then it was possible to update the
+user even if some non-LDAP mapped attributes were changed. This may result in confusing behaviour, when the attribute appeared to be updated,
+but it was not. In the current version, the update is not allowed instead of the successful "lost update" .
```

Comment:
s/, the update is not allowed instead of the successful "lost update"/, the update is not allowed to be performed at all instead to tolerate a successful execution of a "lost update"./g

Reply:
Changing as you suggested. Thanks